client/components/chart/style.scss

```diff
@@ -164,18 +174,21 @@ $y-axis-padding: 0 20px 0 10px;
 
 // Individual bar wrapper & misc
 // 1: Positions the bar in the space as defined by .bar
-// 2: Default value for top so bars grow when they get a new value
-//    (doesn't function correctly for period switching right now, not sure why)
-//    (also doesn't work because updating the DOM is so heavy it stalls all animations)
+// 2: use `transform` (as opposed to top or height) for better transition performance, see: https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/
```

Comment:
Nitpick: this article is somewhat out of date, it would be best to point to a more modern source, such as https://www.html5rocks.com/en/tutorials/speed/high-performance-animations/ . Of course, that risks going stale over time as well. I'm divided on whether we should include a link here; on the one hand it's common knowledge in performance circles, on the other hand performance circles are small and don't include a lot of developers 😄 I'll leave it to your judgement!